Lanchonete/Lanchonete/ Features/Customization/Presentation/CustomizationViewPresenter.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  CustomizationViewPresenter.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 2/9/21.
+//
+
+import Foundation
+
+public protocol CustomizationView: class {
+    func reloadData()
+}
+public class CustomizationViewPresenter {
+    let ingredientManager: IngredientManager
+    let allIngredientManager = AllIngredientsManager()
+    
+    public var view: CustomizationView?
+    
+    init(ingredientManager: IngredientManager) {
+        self.ingredientManager = ingredientManager
+    }
+    // Get all the sandwiches from the menu
+    lazy var ingredients: [Ingredient] = {
+        return allIngredientManager.getIngredientList()
+    }()
+    
+    func numberOfSections() -> Int {
+        return 1
+    }
+    
+    func titleForSection() -> String {
+        return "Avaliable Ingredients"
```

Comment:
```suggestion
        return "Available Ingredients"
```
Tip: you can enable a spell checker in Xcode 😉 
![CleanShot 2021-02-11 at 11 20 28](https://user-images.githubusercontent.com/14120/107630234-27c55680-6c5b-11eb-8983-e78de8c3ad2c.png)

Reply:
Changed this also